packages/storage/src/__tests__/queryExtensions.test.ts

```diff
@@ -58,6 +60,9 @@ describe('Query extensions', () => {
 
   describe('create', () => {
     it('create will save files', async () => {
+      // Mock the fs.mkdir function
+      ;(fs.mkdir as jest.Mock).mockResolvedValue(undefined)
```

Comment:
```suggestion
      // Mock the fs.mkdir function
      vi.mocked(fs.mkdir).mockResolvedValue(undefined)
```

Using `vi` instead of `jest`
Using `vi.mocked` instead of type casting

Reply:
Ah. Thanks. That was AI gen'd and I changed to `vi` but I left that artifact.